### PR TITLE
docs: autoscaler doc for max_scale_up and max_scale_down of target-value strategy

### PR DESCRIPTION
**Pull request by brmzkw**
See https://github.com/hashicorp/nomad-autoscaler/pull/848

### CODE REVIEW

File: website/content/tools/autoscaling/plugins/strategy/target-value.mdx [hunk 1]
```diff
@@ -89,3 +89,11 @@ check "target-value-check" {
 - `threshold` `(float: 0.01)` - Specifies how significant a change in the input
   metric should be considered. Small threshold values can lead to output
   fluctuation.
+
+- `max_scale_up` `(int: unlimited)` - specifies the maximum number of instances
```
Comment (jrasell) [inline website/content/tools/autoscaling/plugins/strategy/target-value.mdx:93]:
```suggestion
- `max_scale_up` `(int: unlimited)` - Specifies the maximum number of instances
```

File: website/content/tools/autoscaling/plugins/strategy/target-value.mdx [hunk 2]
```diff
@@ -89,3 +89,11 @@ check "target-value-check" {
...
+  that can be added in a single scaling operation. It acts as a safeguard
+  against the system scaling up too rapidly.
+
+- `max_scale_down` `(int: unlimited)` - specifies the maximum number of instances
```
Comment (jrasell) [inline website/content/tools/autoscaling/plugins/strategy/target-value.mdx:97]:
```suggestion
- `max_scale_down` `(int: unlimited)` - Specifies the maximum number of instances
```

**Review by jrasell — CHANGES_REQUESTED** [review]
Hi @brmzkw and thanks for raising this. It's looking good from my end, with a couple of minor changes requested.

**Comment by brmzkw** [comment]
fixed :-)

**Review by lgfa29 — APPROVED** [review]
Thanks @brmzkw!

**Review by jrasell — APPROVED** [review]
LGTM, thanks @brmzkw!

**Pull request merged**